src/extension/netcdf-out/src/test/java/org/geoserver/wcs2_0/WCSNetCDFTest.java

```diff
@@ -70,6 +77,10 @@ protected void onSetUp(SystemTestData testData) throws Exception {
         setupRasterDimension(getLayerId(NO2), ResourceInfo.TIME, DimensionPresentation.LIST, null);
         setupRasterDimension(
                 getLayerId(NO2), ResourceInfo.ELEVATION, DimensionPresentation.LIST, null);
+        setupRasterDimension(getLayerId(O3), ResourceInfo.TIME, DimensionPresentation.LIST, null);
+        setupRasterDimension(
+                getLayerId(O3), ResourceInfo.ELEVATION, DimensionPresentation.LIST, null);
+        setupRasterDimension(getLayerId(O3), ResourceInfo.TIME, DimensionPresentation.LIST, null);
```

Comment:
maybe a typo here. It looks like it has already been added at line 80

Reply:
Yep, will remove